### PR TITLE
fix(FEC-11292): when in full screen, button tool tip should state "Exit Full Screen"

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

Issue: When video in full screen, button tooltip state "Full Screen" instead of "Exit Full Screen".
Solution: add tooltip state "Exit Full Screen" when on fullscreen mode .

solves FEC-11292

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: translations/en.i18n.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
       "language": "Language",
       "settings": "Settings",
       "fullscreen": "Fullscreen",
+      "fullscreenExit": "Exit fullscreen",
```
Comment (Yuvalke) [inline translations/en.i18n.json:9]: @JonathanTGold it means we need translations for all

File: src/components/fullscreen/fullscreen.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ import {ButtonControl} from 'components/button-control';
  */
 const mapStateToProps = state => ({
   fullscreen: state.engine.fullscreen,
-  targetId: state.config.targetId
+  targetId: state.config.targetId,
+  isInFullscreen: state.engine.fullscreen
```
Comment (RoyBregman) [inline src/components/fullscreen/fullscreen.js:24]: already exists two lines above

File: translations/en.i18n.json (hunk 1 above)
Comment (RoyBregman) [reply translations/en.i18n.json:9]: Please contact Ron Liberman

File: translations/en.i18n.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
       "pause": "Pause",
       "language": "Language",
       "settings": "Settings",
-      "fullscreen": "Fullscreen",
-      "fullscreenExit": "Exit fullscreen",
+      "fullscreen": "Full screen",
+      "fullscreenExit": "Exit full screen",
```
Comment (yairans) [inline translations/en.i18n.json:9]: pls revert it. Ron L wants to keep 'Fullscreen'

File: src/components/fullscreen/fullscreen.js [hunk 4]
```diff
@@ -19,8 +19,8 @@ import {ButtonControl} from 'components/button-control';
  * @returns {Object} - mapped state to this component
  */
 const mapStateToProps = state => ({
-  fullscreen: state.engine.fullscreen,
-  targetId: state.config.targetId
+  targetId: state.config.targetId,
```
Comment (yairans) [inline src/components/fullscreen/fullscreen.js:22]: no need to change it

**Review by yairans — CHANGES_REQUESTED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**